tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/Emitter.scala

```diff
@@ -108,6 +111,9 @@ final class Emitter(semantics: Semantics, outputMode: OutputMode) {
     }
   }
 
+  def emitCustomFooter(customFooter: String, builder: JSFileBuilder): Unit =
+    emitLines(customFooter, builder)
+
```

Comment:
Not really, but @sjrd advised on it because at some point we will have to factor out some common `ScalaJSOptimizer` and `ScalaJSClosureOptimizer`.